source/dem/force_chains_visualization.cc

```diff
@@ -91,8 +87,21 @@ ParticlesForceChains<dim, contact_model, rolling_friction_model>::
                      MPI_Comm                        mpi_communicator,
                      const std::string               folder,
                      const unsigned int              iter,
-                     const double                    time)
+                     const double                    time,
+                     typename dem_data_structures<dim>::adjacent_particle_pairs
+                       &local_adjacent_particles,
+                     typename dem_data_structures<dim>::adjacent_particle_pairs
+                       &ghost_adjacent_particles)
 {
+  // Creating containers
+  std::vector<Point<3>> vertices;
+  std::vector<double>   normal_forces_vector;
```

Comment:
I think you need to give those a defaul value otherwise if you have a processor with nothing won't you get an error ? LIke if the vectors are empty